src/deployment/registration.py

```diff
@@ -275,7 +277,7 @@ def create_application_registration(
 def add_application_password(
     app_object_id: UUID, subscription_id: str
 ) -> Tuple[str, str]:
-    def create_password() -> Tuple[str, str]:
+    def create_password(data: Any) -> Tuple[str, str]:
```

Comment:
Does `data` really need to be `Any` in this position?

Reply:
yes mypy asks for a type definition here